web_src/js/features/repo-branch.js

```diff
@@ -16,3 +29,15 @@ export function initRepoBranchButton() {
     $($(this).attr('data-modal')).modal('show');
   });
 }
+
+function initRepoRenameBranchButton() {
+  const showRenameBranchModal = $('.show-rename-branch-modal');
```

Comment:
There is already "show-modal" code, can it be re-used?

Reply:
done